pool.go

```diff
@@ -0,0 +1,31 @@
+package t1k
+
+import "time"
+
+// Pool 基本方法
+type Pool interface {
+	// 获取资源
+	Get() (interface{}, error)
+	// 资源放回去
+	Put(interface{}) error
+	// 关闭资源
+	Close(interface{}) error
+	// 释放所有资源
+	Release()
+	// 返回当前池子内有效连接数量
+	Len() int
+}
+
+// PoolConfig 连接池相关配置
+type PoolConfig struct {
+	//连接池初始连接数
```

Comment:
在 `pool.go` 中，`PoolConfig` 结构体的字段缺少注释。建议为每个字段添加注释，说明其用途。

```suggestion
type PoolConfig struct {
	// InitialCap 连接池初始连接数
	InitialCap int
	// MaxCap 最大并发存活连接数
	MaxCap int
	// MaxIdle 最大空闲连接
	MaxIdle int
	// Factory 连接工厂
	Factory ConnectionFactory
	// IdleTimeout 连接最大空闲时间，超过该时间则将失效
	IdleTimeout time.Duration
}